tools/deployment/make_linux_package.sh

```diff
@@ -268,6 +268,7 @@ function main() {
     BINARY_DEBUG_DIR=$DEBUG_PREFIX/usr/lib/debug/usr/bin/
     mkdir -p $BINARY_DEBUG_DIR
     cp "$BUILD_DIR/osquery/osqueryd" "$BINARY_DEBUG_DIR/osqueryd.debug"
+    strip --only-keep-debug "$BINARY_DEBUG_DIR/osqueryd.debug"
```

Comment:
Can you add this line to https://github.com/facebook/osquery/pull/4277/files#diff-927382df658750d5c4e121f6f048470bR249 too?

Reply:
Done.